### PR TITLE
script.js: Add error message alert

### PR DESCRIPTION
**Pull request by ofemeteng**
This adds an error message alert to the page when an invalid txt file is provided

Closes https://github.com/prabhakar267/WA-Reader/issues/60

### CODE REVIEW

File: static/js/script.js [hunk 1]
```diff
@@ -66,7 +66,14 @@ function uploadFiles(event){
...
             } else {
-                show_error_message(response.error_message);
+                $('.upload-popup').prepend('<div class="alert alert-danger alert-dismissible">' +
+                '<button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>' +
+                response.error_message + 
+                '</div>');
+
+                setTimeout(function() {
+                    $('.alert').fadeOut('slow');
+                }, 4000);
```
Comment (prabhakar267) [inline static/js/script.js]: Why did you add this here? Can you see the problem with `show_error_message` function, since there are other scenarios as well which can raise error.

Comment (ofemeteng) [reply static/js/script.js]: The thing is in index.html, you passed in a reference to displace an error message if a message is set or not.

However, you are using Jinja syntax to determine that and not pure JavaScript, so in reality, the error is only set if it is passed in from app.py and not when there is an error with asynchronous API calls by JavaScript

Comment (prabhakar267) [reply static/js/script.js]: There's a method in `script.js` named `show_error_message`, you can fix/modify that method instead of adding the code here

Comment (ofemeteng) [reply static/js/script.js]: Ok. I'll have a look

**Comment by prabhakar267** [comment]
@ofemeteng Checkout #64, it provides a more generic solution. 
So I'm closing this. Thanks for contribution though
